runtime/plugins/linter/linter.lua

```diff
@@ -82,6 +82,7 @@ function preinit()
     makeLinter("pyflakes", "python", "pyflakes", {"%f"}, "%f:%l:.-:? %m")
     makeLinter("mypy", "python", "mypy", {"%f"}, "%f:%l: %m")
     makeLinter("pylint", "python", "pylint", {"--output-format=parseable", "--reports=no", "%f"}, "%f:%l: %m")
+    makeLinter("ruff", "python", "ruff", {"check", "--output-format=pylint", "%f"}, "%f:%l: %m")
```

Comment:
You could get the column too by using `--output-format=concise`, which allows micro to put a little underline on the exact column of the error:
```suggestion
    makeLinter("ruff", "python", "ruff", {"check", "--output-format=concise", "%f"}, "%f:%l:%c: %m")
```